util/winutil/winutil_windows.go

```diff
@@ -148,6 +152,9 @@ func DeleteRegValue(name string) error {
 
 func deleteRegValueInternal(subKey, name string) error {
 	key, err := registry.OpenKey(registry.LOCAL_MACHINE, subKey, registry.SET_VALUE)
+	if err == registry.ErrNotExist {
```

Comment:
This changes the semantics, it would have returned an error. Presumably the caller already suppressed errors, does the caller need to pay attention to whatever other errors might be returned by line 160?

Reply:
The two places it's used actually do check for errors and will abort their cleanups if there is an error. I think this actually improves their semantics since in a cleanup it's fine if the thing being deleted doesn't exist.